docs/skinning.md

```diff
@@ -0,0 +1,111 @@
+# Skinning Your uPortal-home #
+
+_Prerequisites:_
+
+_A working uPortal-app-framework application._
+
+_A code editor._
+
+For a more in-depth discussion of theming, see https://github.com/UW-Madison-DoIT/uw-frame/blob/master/docs/theming.md
+
+1. Locate and open the frame-config.js file in your uw-frame-components/js/ directory.
+
+
+2. Increment the themeVersion in line 9. 
```

Comment:
Thanks for the clarification @Doug-Reed and @davidmsibley! :+1: 
I was unsure of the reasoning just looking at code comments and usages of `themeVersion`.
I have some more follow up questions on that, but don't want to derail the code review here.
Where is the best place to ask questions, like this, about uw-frame?